codeforamerica/michigan-benefits
# Create PDF with Snap app info after signing

**WHY**: So we can fax it in.

To test:
* visit home page
* click "apply now"
* go through flow
* open file named "test_pdf.pdf" and confirm that data you enterd is
  included on page 9
